web/src/views/InvestigationDetail.vue

```diff
@@ -212,20 +221,19 @@ export default defineComponent({
       ));
     }
 
+    function getSelectedFrameStyle(): BandSpec[] {
+      return frames.value.filter((frame: TiffFrame) => frame.displayed).map((frame: TiffFrame) => ({
+        frame: frame.frame,
+        palette: ['#000', `#${frame.color}`],
```

Comment:
If we want to reduce the length/size of the style, I think we can make this just:

```suggestion
        palette: `#${frame.color}`,
```